test/integration/status_test.lua

```diff
@@ -0,0 +1,238 @@
+local t = require('luatest')
+local g = t.group()
+
+local fio = require('fio')
+
+local Capture = require('luatest.capture')
+
+local helper = require('test.helper')
+local cmd = helper.cartridge_cmd
+
+local TEST_APP_NAME = 'test_app'
+
+local RUN_DIR = fio.pathjoin(helper.tempdir, 'test_run')
+local TEST_APP_DIR = fio.pathjoin(helper.tempdir, TEST_APP_NAME)
+
+local RUN_DIR_OPT = {'--run-dir', RUN_DIR}
+
+local running_pids = {}
+
+g.before_each(function()
+    fio.rmtree(RUN_DIR)
+    fio.rmtree(TEST_APP_DIR)
+    fio.copytree(
+        fio.pathjoin(fio.cwd(), 'test/integration/test_app'),
+        TEST_APP_DIR
+    )
+    running_pids = {}
+end)
+
+g.after_each(function()
+    for _, pid in pairs(running_pids) do
+        helper.kill_process(pid)
+    end
+end)
+
+local function check_is_running(instance_fullname)
```

Comment:
Could you please move it into helpers (https://github.com/tarantool/cartridge-cli/blob/master/test/integration/start_stop_stateboard_only_test.lua#L36) ?

As it was done e.g. here - https://github.com/tarantool/cartridge/blob/master/test/helper.lua